x/gov/codec.proto

```diff
@@ -224,7 +231,8 @@ message TallyMsg {
 // with a reference to the electorate that approved it
 message TextResolutionMsg {
   weave.Metadata metadata = 1;
-  string resolution = 2;
+  bytes electorate_id = 2 [(gogoproto.customname) = "ElectorateID"];
```

Comment:
This message cannot contain this ref.
The ref is stored in the Proposal, and is the Electorate that actually voted on it.
This one added here is arbitrary.

Hey devs, let's hold an election, stating that the board of directors said "all devs get a pay raise".

Clearly not good. Now, if it recorded that all devs stated "all devs get a pay raise", fair enough, as they said it.

Reply:
So what you are saying is that we are to use proposal to fetch the electorate ref for Resolution? 
Fair